libs/sdk-ui-kit/src/List/ListItem.tsx

```diff
@@ -165,7 +165,7 @@ export class SingleSelectListItem extends Component<ISingleSelectListItemProps,
 
         return (
             <div role="item-info" className="gd-list-item-bubble s-list-item-info">
-                <BubbleHoverTrigger tagName="div" showDelay={200} hideDelay={0}>
+                <BubbleHoverTrigger tagName="div" showDelay={200} hideDelay={100} eventsOnBubble={true}>
```

Comment:
This change will affect another feature so I would like to introduce a property for this component to handle your expectation.

WDYT?